document-store/src/main/java/org/hypertrace/core/documentstore/postgres/query/v1/vistors/PostgresFunctionExpressionVisitor.java

```diff
@@ -50,7 +51,7 @@ public String visit(final FunctionExpression expression) {
     if (numArgs == 1) {
       String parsedExpression = getParsedExpression(expression.getOperands().get(0));
       return expression.getOperator().equals(FunctionOperator.LENGTH)
-          ? String.format("ARRAY_LENGTH( %s, 1 )", parsedExpression)
+          ? String.format("ARRAY_LENGTH( %s, %s )", parsedExpression, ARRAY_DIMENSION)
```

Comment:
%d for int

Reply:
will do in follow-up, using %s rest all the places.